### PR TITLE
Add blog post about release of autoupgrade v6.1 and v6.2

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | This PR adds the announcement of the latest release of autoupgrade module.
| Fixed ticket? | /

### CODE REVIEW

File: content/news/posts/2024/autoupgrade-v6.1-release.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Autoupgrade module v6.1.0 is Available (1-Click Upgrade)"
+subtitle: "We are pleased to introduce the new minor version release of the Autoupgrade module, also known as 1-Click Upgrade."
+date:   2024-09-12
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6.1-release.md]:
```suggestion
date:   2024-09-17
```

File: content/news/posts/2024/autoupgrade-v6.1-release.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+In the footsteps of the latest module releases ([see version 6.0.0 release blogpost](https://build.prestashop-project.org/news/2024/autoupgrade-v6-release/)), this version brings a number of improvements to streamline PrestaShop updates. Explore the changelog below to find out what version 6.1.0 brings to the upgrade module.
+
+![1-click upgrade module logo](/assets/images/2024/07/autoupgrade-logo.png)
+
+### What’s inside
+
+#### Global improvements:
+
+* **Version comparison** - Fix an issue where the number of altered or deleted core files is always zero.
+* **UX optimization** - Improve user experience during update & restore processes with newer to-do lists & simplification of UI.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6.1-release.md]:
```suggestion
* **UX optimization** - Improve user experience during update and restore processes with newer to-do lists and simplification of UI.
```

**Review by coffrep — APPROVED** [review]
Hi @kpodemski thanks for pinging me. Great post @Quetzacoalt91, I've just changed the date and added a minor edit. Approving with edits on my end!

**Comment by Quetzacoalt91** [comment]
Thanks @coffrep, the date has been changed and I'll think about the `&` next time.

**Comment by kpodemski** [comment]
hey @Quetzacoalt91 

maybe you'd like to mention 6.2 as well?

**Comment by Quetzacoalt91** [comment]
As this is only a bug fix, I can drop a line about it

**Comment by kpodemski** [comment]
@Quetzacoalt91 great, we'll publish it tomorrow :)

File: content/news/posts/2024/autoupgrade-v6.1-release.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+#### For developers:
+
+* **New translation in Crowdin** - New module translations added to Crowdin, following its recent integration, in different languages.
+* **Translation availability in Storybook** - New feature to translate the module interface in Storybook User Interface tool.
+* **Multiple tech improvements** - JS format standardization, JS error handler, local archive check, new logs exception… Many corrections and tech optimizations have been added in this version.
+
+
+The versions 6.1.0 and 6.2.0 of the module also brings other new improvements, you can read the full changelog by clicking on the links below.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6.1-release.md]:
```suggestion
The versions 6.1.0 and 6.2.0 of the module also bring other new improvements, you can read the full changelog by clicking on the links below.
```

**Review by coffrep — APPROVED** [review]
Hi @Quetzacoalt91 , thanks for the edit, I just added one suggestion and approving. Thanks!

**Comment by Quetzacoalt91** [comment]
* Merged v6.1 changelog with v6.2.
* Updated release date for tomorrow

File: content/news/posts/2024/autoupgrade-v6.1-release.md (hunk 3 above)
Comment (Quetzacoalt91) [reply content/news/posts/2024/autoupgrade-v6.1-release.md]: Thanks

**Comment by kpodemski** [comment]
@Quetzacoalt91 I think it's ready and we can publish it, right? please let me know because I see a WIP label

**Comment by coffrep** [comment]
Hi @Quetzacoalt91 and @kpodemski good for me, ready for publishing!

**Comment by Quetzacoalt91** [comment]
I forgot to take the label off. It's ready on my side as well :+1:

**Review by kpodemski — APPROVED** [review]

**Pull request merged**